app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_split = command.split()
```

Comment:
The input command should be validated to ensure it contains exactly three elements and starts with 'mv'. This will prevent incorrect command formats from causing issues.